recipes/econml/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "econml" %}
+{% set version = "0.8.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/econml-{{ version }}.tar.gz
+  sha256: fa13815f275729c49a0e69f9ab691d1d9dbbb91ba799647419abd6ab93abb9f8
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python
+    - sphinx <3.2
+    - sphinx_rtd_theme
+  run:
+    - python-graphviz
+    - joblib >=0.13.0
+    - keras <2.4
+    - matplotlib-base
+    - numba !=0.42.1
+    - numpy
+    - pandas <1.1
+    - python
+    - scikit-learn >0.21.0
+    - scipy !=1.4.0
+    - sparse
+    - statsmodels >=0.9
+    - tensorflow >1.10,<2.3
+
+test:
+  imports:
+    - econml
+    - econml.data
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Microsoft/EconML
+  summary: This package contains several methods for calculating Conditional Average Treatment Effects
+  license: Apache-2.0
+  license_file: .eggs/Sphinx-3.1.2-py3.7.egg/EGG-INFO/LICENSE
```

Comment:
That looks wrong. Can you copy and include https://github.com/microsoft/EconML/blob/master/LICENSE in the directory?